docs/source/models/concept_models/vivarium_mncnh_portfolio/ai_ultrasound_module/module_document.rst

```diff
@@ -232,15 +228,15 @@ Estimated gestational age should be calculated by adding a randomly sampled valu
   * - Output
     - Value
     - Note
-  * - A. Type of ultrasound received
-    - *none* / *standard* / *AI-assisted*
-    - Used for V&V and for estimation of output B
-  * - B. Estimated gestational age
+  * - Ultrasound type
```

Comment:
Hope this isn't controversial, but I removed the lettering from module outputs because a) it's redundant to the statements of where to record above and a) it makes editing these more annoying, it's easy to end up with two (a)s